manager/src/test/java/com/bakdata/quick/manager/topic/KafkaTopicServiceTest.java

```diff
@@ -131,7 +132,7 @@ void shouldNotCreateTopicThatAlreadyExists() {
         final TopicService topicService = this.newTopicServiceForAvro();
         this.setupSuccessfulMock();
 
-        final TopicCreationData requestData = new TopicCreationData(TopicWriteType.MUTABLE, null, null, null);
+        final TopicCreationData requestData = new TopicCreationData(TopicWriteType.MUTABLE, null, null, null, true,null);
```

Comment:
```suggestion
        final TopicCreationData requestData = new TopicCreationData(TopicWriteType.MUTABLE, null, null, null, true, null);
```

For all changed calls in this class